### PR TITLE
Change unsigned longs to uint64_ts to prevent OOB memory writes in WebVR lib

### PR DESCRIPTION
**Pull request by jaadelgren**
Change `unsigned long`s to `uint64_t`s to prevent OOB memory write in library_vr.js. Could also change to `uint32_t`s and modify library_vr.js, which I'm open to doing if that's deemed preferable.

### CODE REVIEW

**Comment by kripken** [comment]
Thanks!

I think we should switch to 32-bit ints, as that's more efficient, unless there's a reason 64 bit precision is needed here?

Also please add yourself to AUTHORS.

**Comment by jaadelgren** [comment]
Cool, I can do that. It looks like another, similar pull request was created for this issue as well. https://github.com/emscripten-core/emscripten/pull/8186

**Comment by jaadelgren** [comment]
Oh and yes, I agree. I can't really see a reason for 64-bit precision in this case.

File: AUTHORS [hunk 1]
```diff
@@ -386,4 +386,5 @@ a license to everyone to use it as detailed in LICENSE.)
 * Shane Peelar <lookatyouhacker@gmail.com>
 * Alessandro Pignotti <alessandro@leaningtech.com>
 * Zheng Tao Lee <zhengtao.lee@autodesk.com> (copyright owned by Autodesk, Inc.)
-* Martina Kraus <kraus.martina.m@googlemail.com>
\ No newline at end of file
+* Martina Kraus <kraus.martina.m@googlemail.com>
+* Jacob Adelgren <jake@eevo.com>
```
Comment (sbc100) [inline AUTHORS]: Can you add newline at the end of the file?

**Comment by kripken** [comment]
Great, thanks @jaadelgren!

**Comment by jaadelgren** [comment]
Sure thing :)

**Pull request merged**